kola/tests/etcd/discovery.go

```diff
@@ -23,12 +23,12 @@ import (
 	"github.com/coreos/mantle/platform"
 )
 
-func DiscoveryV2(cluster platform.Cluster) error {
-	return discovery(cluster, 2)
+func DiscoveryV2(c platform.Cluster, _ func(string, platform.Machine) error) error {
```

Comment:
That'd be much nicer but I can't figure out how to do that without causing an import cycle.

The kola package requires tests call `Register()` inside the kola package to glue in the separate test packages. If I pass a named type from kola in as an argument in the Test.Run function then the tests will have a signature like `func Tester(c platform.Cluster, nr kola.NativeRunner) error {}` when defined outside the kola package causing the import cycle.

I could name the type as a platform.NativeRunner. Does that sound reasonable?